conformance/tests/directives_no_type_check.py

```diff
@@ -0,0 +1,19 @@
+"""
+Tests the typing.no_type_check decorator.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/directives.html#no-type-check
+
+from typing import no_type_check
+
+# > To mark portions of the program that should not be covered by type hinting,
+# > you can use the @typing.no_type_check decorator on a class or function.
+# > Functions with this decorator should be treated as having no annotations.
+
+
+@no_type_check
+def func1(a: int, b: str) -> None:
+    c = a + b  # No error should be reported
+    return 1  # No error should be reported
+
+
```

Comment:
We should check use of this function as well, e.g. `func(b"invalid", b"arguments")` should not error

Reply:
Added test as suggested.